### PR TITLE
Add compatibility information to sfr_box

### PR DESCRIPTION
**Pull request by epenet**
## Proposed change

SSIA
Linked to https://github.com/home-assistant/core/issues/106828

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by epenet** [comment]
cc @DocJackson / @Danifly80

File: source/_integrations/sfr_box.markdown [hunk 1]
```diff
@@ -26,3 +26,9 @@ This integration provides the following platforms:
 - Sensors - such as ADSL line status, attenuation, noise and data rate.
 
 {% include integrations/config_flow.md %}
+
+## compatibility
```
Comment (c0ffeeca7) [inline source/_integrations/sfr_box.markdown]:
```suggestion
## Compatibility
```

File: source/_integrations/sfr_box.markdown [hunk 2]
```diff
@@ -26,3 +26,9 @@ This integration provides the following platforms:
...
+
+The integration uses the REST API, which is known to be available on models `NB4`, `NB5`, `NB6`, `NB6V` and `NB6VAC`.
```
Comment (c0ffeeca7) [inline source/_integrations/sfr_box.markdown]:
```suggestion
The integration uses the REST API, which is known to be available on models `NB4`, `NB5`, `NB6`, `NB6V`, and `NB6VAC`.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @epenet :+1:

**Pull request merged**